cypher/cypher-docs/src/docs/dev/ql/administration/constraints/index.asciidoc

```diff
@@ -252,9 +252,70 @@ include::drop-a-constraint.asciidoc[leveloffset=+2]
 
 include::drop-a-non-existing-constraint.asciidoc[leveloffset=+2]
 
+
 [[administration-constraints-list-constraint]]
 === List constraints
-include::list-constraints.asciidoc[leveloffset=+2]
+
+Listing all constraints can be done with `SHOW CONSTRAINTS`, which will produce a table with the following columns:
+
+.List constraints output
+[options="header", width="100%", cols="1a,4,^.^,^"]
+|===
+| Column
+| Description
+| Brief output
+| Verbose output
+
+| `id`
+| The id of the constraint.
+| `+`
+| `+`
+
+| `name`
+| Name of the constraint (explicitly set by the user or automatically assigned).
+| `+`
+| `+`
+
+| `type`
+| The ConstraintType of this constraint (`UNIQUENESS`, `NODE_PROPERTY_EXISTENCE`, `NODE_KEY`, or `RELATIONSHIP_PROPERTY_EXISTENCE`).
+| `+`
+| `+`
+
+| `entityType`
+| Type of entities this constraint represents (nodes or relationship).
+| `+`
+| `+`
+
+| `labelsOrTypes`
+| The labels or relationship types of this constraint.
+| `+`
+| `+`
+
+| `properties`
+| The properties of this constraint.
+| `+`
+| `+`
+
+| `ownedIndexId`
+| The id of the index associated to the constraint, or `null` if no index is associated with the constraint.
+| `+`
+| `+`
+
+| `options`
+| The options passed to `CREATE` command, for the index associated to the constraint, or `null` if no index is associated with the constraint.
+|
+| `+`
+
+| `createStatement`
+| Statement used to create the constraint.
+|
+| `+`
+|===
+
+[NOTE]
+Note that the deprecated built-in procedures for listing constraints, such as `db.constraints`, work as before and are not affected by the <<administration-security-administration-database-constraints, `SHOW CONSTRAINTS` privilege>>.
```

Comment:
```suggestion
The deprecated built-in procedures for listing constraints, such as `db.constraints`, work as before and are not affected by the <<administration-security-administration-database-constraints, `SHOW CONSTRAINTS` privilege>>.
```

To make it consistent with the Index one

Reply:
Of course! Thanks for spotting this!

Reply:
Also since we move the table of columns in here, shouldn't that mean we can remove the columns file?